karma.conf.js

```diff
@@ -29,20 +29,16 @@ module.exports = (config) => {
             esm: {
                 nodeResolve: true,
             },
-            browsers: [
-                path.resolve(path.join(__dirname, 'scripts/firefox.sh')),
-                path.resolve(path.join(__dirname, 'scripts/chrome.sh')),
-            ],
+            browsers: ['FirefoxHeadless'],
```

Comment:
I'm curious why this change is in this PR. I wonder if you could give some background on this?

Reply:
It might not always be apparent when running `yarn test`, but until this PR we have been actually running the tests of Chrome twice and Firefox once. This is due to the fact that the `@open-wc` tooling assigned Chrome by default and then we were subsequently assigned Chrome and Firefox (on top of that) in these settings.

The custom scripts we were using were made for the project at a time when WSL stored its browsers in a non-normative location so without them, we couldn't run tests on local machines using that OS without them. More recent releases of WSL have returned access to the browsers in their expected locations so removing them has been on my list for a while.

I can't fully speak to the reason why it was needed in this specific PR, but due to some of the updates in the versions of various Babel dependencies, the tests stop running appropriately on CI when being run in the browsers that were discovered with these scripts. I wish I had a couple of levels up of CI skills and I could fully debug what was going on there, but when using the more standard browser accessors fixed the issue I decided to be OK with it. Running on two browsers, instead of these, might also give us a tiny speed benefit, though they were running mostly in parallel anyways. 

Right now, I'm doing a small test to discover which of the various configurations for acquiring Firefox is the best in that I can consistently run the tests with no trouble of CI and my personal machine, but my work machine doesn't like testing focus based interactions with it, which was one of the main reasons to move to headless to begin with.

I'm hacking through a last

Reply:
Thanks for that. I looks like we should merge this up and follow up on the testing in a future PR.